internal/services/iot/routes_test.go

```diff
@@ -86,7 +86,7 @@ func TestAccRoute_S3(t *testing.T) {
 	}
 	tt := acctest.NewTestTools(t)
 	defer tt.Cleanup()
-	bucketName := sdkacctest.RandomWithPrefix("test-acc-scaleway-iot-route-s3")
+	bucketName := sdkacctest.RandomWithPrefix("tf-tests-scaleway-iot-route-s3")
```

Comment:
not the correct prefix I think. Could you extract the prefix and reference it here?

Reply:
https://github.com/scaleway/terraform-provider-scaleway/blob/master/internal/services/object/testfuncs/sweep.go#L36

Reply:
In that case, I would suggest changing testSweepStorageObjectBucket to use IsTestResource instead of strings.HasPrefix